nvim/after/ftplugin/tex.lua

```diff
@@ -1,23 +1,19 @@
-local M = {}
-
-local config = require("nvim-surround.config")
-
 require("nvim-surround").buffer_setup({
   surrounds = {
     ["e"] = {
       add = function()
-        local env = require("nvim-surround.config").get_input "Environment: "
+        local env = require("nvim-surround.config").get_input("Environment: ")
         return { { "\\begin{" .. env .. "}" }, { "\\end{" .. env .. "}" } }
       end,
     },
     ["Q"] = {
       add = { "``", "''" },
-      find = "%b``''",
+      find = "``.-''",
```

Comment:
Here we needed a `.-` between the backticks and single quotes to match all characters between the left and right pair. You can think of it as being the same as a traditional regex's `.*` match, but in this case `.-` is used for lazy matching instead of greedy matching.